common/src/main/java/com/wynntils/commands/QuestCommand.java

```diff
@@ -0,0 +1,154 @@
+                                .executes(this::trackQuest)))
+                .then(Commands.literal("untrack").executes(this::untrackQuest))
+                .then(Commands.literal("wiki")
+                        .then(Commands.argument("quest", StringArgumentType.greedyString())
+                                .suggests(QUEST_SUGGESTION_PROVIDER)
+                                .executes(this::lookupOnWiki)))
+                .executes(this::syntaxError);
+    }
+
+    private int listQuests(CommandContext<CommandSourceStack> context) {
+        List<QuestInfo> quests = QuestBookModel.getQuests();
+        // FIXME: sort argument "level", "distance", "alphabetical"
+        // FIXME: filter "started", "can start", etc
+
+        MutableComponent response = new TextComponent("All known quests:").withStyle(ChatFormatting.AQUA);
+
+        for (QuestInfo quest : quests) {
+            response.append(new TextComponent("\n - ").withStyle(ChatFormatting.GRAY))
+                    .append(new TextComponent(quest.getName())
+                            .withStyle(style -> style.withClickEvent(new ClickEvent(
+                                    ClickEvent.Action.RUN_COMMAND,
+                                    "/quest info " + quest.getName().replaceAll(" ", " "))))
+                            .withStyle(style -> style.withHoverEvent(
+                                    new HoverEvent(HoverEvent.Action.SHOW_TEXT, new TextComponent("Click for info"))))
+                            .withStyle(ChatFormatting.WHITE));
+        }
+
+        context.getSource().sendSuccess(response, false);
+        return 1;
+    }
+
+    private int questInfo(CommandContext<CommandSourceStack> context) {
+        String questName = context.getArgument("quest", String.class).toLowerCase(Locale.ROOT);
+        List<QuestInfo> matchingQuests = QuestBookModel.getQuests().stream()
+                .filter(questInfo ->
+                        questInfo.getName().toLowerCase(Locale.ROOT).contains(questName))
+                .toList();
+
+        if (matchingQuests.size() < 1) {
+            context.getSource()
+                    .sendFailure(
+                            new TextComponent("Quest '" + questName + "' not found").withStyle(ChatFormatting.RED));
+            return 0;
+        } else if (matchingQuests.size() > 1) {
+            MutableComponent error =
+                    new TextComponent("Quest '" + questName + "' match several quests:").withStyle(ChatFormatting.RED);
+            for (var quest : matchingQuests) {
+                error.append(new TextComponent("\n - ").withStyle(ChatFormatting.GRAY))
+                        .append(new TextComponent(quest.getName()).withStyle(ChatFormatting.WHITE));
+            }
+            context.getSource().sendFailure(error);
+            return 0;
+        }
+
+        QuestInfo quest = matchingQuests.get(0);
+
+        MutableComponent response =
```

Comment:
It will be so much fun to I18n commands. (This also applies to almost every other command class though..)

Reply:
Not sure we need to do that, though. If we consider commands to be "advanced user" thing, it can probably stay english. (Don't know if you ever run like a localized Linux, and just gotten so annoyed at error messages being incomprehensible in your native language instead of English :-))